grinder/top_ports.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+class TopPorts:
```

Comment:
Maybe it is better to put this class into the defaultvalues.py? In the end

Reply:
This class is moved to defaultvalues.py